src/Analyzers/CSharp/Tests/UseImplicitOrExplicitType/UseExplicitTypeTests.cs

```diff
@@ -362,6 +362,62 @@ void Method(int? x)
             await TestInRegularAndScriptAsync(before, after, options: ExplicitTypeExceptWhereApparent());
         }
 
+#if !CODE_STYLE // TODO: Skipped tests in CodeStyle layer depend on new compiler API (IsNativeIntegerType) that is not available in CodeStyle layer.
```

Comment:
:memo: there is a similar region below for tests involving new nullability APIs

Reply:
Thanks, yes this is a know drawback of shared code between CodeStyle package based on earlier CodeAnalysis version and Features layer based on latest Roslyn. I plan to bring this up for discussion at next IDE design meeting on how to handle these changes in future.

Reply:
can we explicitly update https://github.com/dotnet/roslyn/issues/41462 to mention nint/nuint?

Reply:
it can link to this pr btw.

Reply:
Added a note in https://github.com/dotnet/roslyn/issues/41462 with link to this PR